datadog_checks_dev/datadog_checks/dev/tooling/commands/env/start.py

```diff
@@ -214,6 +243,13 @@ def start(ctx, check, env, agent, python, dev, base, env_vars):
             environment.update_check()
             echo_success('success!')
 
+    if profile_memory and on_ci:
+        environment.metadata['sampling_start_time'] = time.time()
+
+        echo_waiting('Updating metadata... '.format(env), nl=False)
+        environment.write_config()
+        echo_success('success!')
```

Comment:
Maybe I'm missing something but:

1. Should `if profile_memory and on_ci:` be put before this:

https://github.com/DataDog/integrations-core/blob/5d0f8c7c3b8629b2ede6a29c33372aed612fa521/datadog_checks_dev/datadog_checks/dev/tooling/commands/env/start.py#L168-L170

2. And also avoid duplicating this ?

```
        echo_waiting('Updating metadata... '.format(env), nl=False)
        environment.write_config()
        echo_success('success!')
```

Reply:
No because we want the sampling time to start at the end so things like starting the Agent, running extra start-up commands, etc. are not included. I added a comment